### DISCUSSION TITLE
How to create GDPS at Google Cloud?

### DISCUSSION THREAD

**Discussion by Iced-Coded** [Cvolton/GMDprivateServer#759] [General]
idk what to tell, everything is at name of

**Comment by ghost** [comment]
**You need a valid phone number to sign up to Google Cloud**
Sign up for Google Cloud at [https://cloud.google.com/](https://cloud.google.com/) and you can click "Get started for free". There are tutorials at [https://cloud.google.com/php/](https://cloud.google.com/php/) on how to run a PHP website. Instead of a website, upload this repository to your server and it should work once you setup the webserver. After setting up the webserver, please look for tutorials on how to setup a GDPS and creating a database on Google Cloud.
**I only use Google Cloud for voice recognition, so I don't know their server hosting very well.**

**Reply by ghost** [reply]
I personally use HostGator for PHP servers and databases, I don't know if it works on HostGator, but I will try very soon
